### PR TITLE
add id_ID Color

### PR DESCRIPTION
**Pull request by cacing69**
i add some color for id_ID provider

### CODE REVIEW

**Comment by cacing69** [comment]
bump @fzaninotto

File: src/Faker/Provider/id_ID/Color.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Created by PhpStorm.
```
Comment (OskarStark) [inline src/Faker/Provider/id_ID/Color.php]: Please remove this phpdoc

Comment (cacing69) [reply src/Faker/Provider/id_ID/Color.php]: aaah i am sorry, i forget it, will remove it

Comment (cacing69) [reply src/Faker/Provider/id_ID/Color.php]: done i have it remove it

**Comment by cacing69** [comment]
@localheinz may i include link source for this color ?

**Comment by localheinz** [comment]
@cacing69 

That would be helpful!

**Comment by cacing69** [comment]
@localheinz yeaah i have added link on source

File: src/Faker/Provider/id_ID/Color.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+     * @link https://id.wikipedia.org/wiki/Daftar_warna
+     **/
+    protected static $safeColorNames = array(
+        'abu-abu', 'biru', 'biru dongker', 'biru laut', 'coklat', 'emas', 'hijau', 'hitam', 'kuning',
+        'magenta', 'mawar', 'merah', 'merah marun', 'merah jambu', 'nila', 'jingga', 'perak', 'putih', 'ungu',
+        'cokelat',
+    );
+    
+    /**
+     * @link http://listeningsection.blogspot.com/2017/08/100-nama-warna-dalam-bahasa-inggris.html
```
Comment (fzaninotto) [inline src/Faker/Provider/id_ID/Color.php]: the content on this site is copyrighted. Please find a datasource compatible with Faker's MIT license

File: src/Faker/Provider/id_ID/Color.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+     * @link https://www.yec.co.id/inggris/nama-nama-warna-dalam-bahasa-inggris/
+     **/
+    protected static $allColorNames = array(
+        'Merah Indian', 'Koral Terang', 'Salmon', 'Salmon Gelap', 'Salmon Terang', 'Krimson', 'Merah',
```
Comment (fzaninotto) [inline src/Faker/Provider/id_ID/Color.php]: please order the items alphabetically, it eases detection of duplicates and the understanding of future diffs.

Comment (Slamdunk) [reply src/Faker/Provider/id_ID/Color.php]: What about adding a unit test for this purpose?

Comment (fzaninotto) [reply src/Faker/Provider/id_ID/Color.php]: overkill IMO

File: src/Faker/Provider/id_ID/Color.php (hunk 2 above)
Comment (cacing69) [reply src/Faker/Provider/id_ID/Color.php]: sorry for my mistake, now i fix it use data from wikipedia, but some data i change with manual translate

File: src/Faker/Provider/id_ID/Color.php (hunk 3 above)
Comment (cacing69) [reply src/Faker/Provider/id_ID/Color.php]: i done for fix order item.

**Comment by cacing69** [comment]
@rslhdyt  bump

**Comment by rianbagus** [comment]
> i add some color for id_ID provider

keep trying, and give the best

**Comment by ovieinten** [comment]
what a good innovation!

**Comment by rslhdyt** [comment]
@cacing69 Ok Mas

**Comment by cacing69** [comment]
@rslhdyt thank you mas 😄

**Comment by salihinlandrex** [comment]
I confirm that this is true, can someone approve

**Comment by fzaninotto** [comment]
Thanks!

**Pull request merged**